Makefile

```diff
@@ -1,3 +1,8 @@
+PWD=$(shell pwd)
+export C_INCLUDE_PATH=${PWD}/extern/tmp
+export LIBRARY_PATH=${PWD}/extern/tmp
+export LD_LIBRARY_PATH=${PWD}/extern/tmp
+
```

Comment:
so bad, i am sorry for that i forgot to commit bootstrap.sh.
in bootstrap.sh,  it will call setup.sh to make ./extern/tmp dir.
i don't know what will happen when we have already installed rocksdb, and i will make a test.